ra/ra_test.go

```diff
@@ -1185,26 +1184,18 @@ func TestNewCertificate(t *testing.T) {
 	_ = createFinalizedAuthorization(t, sa, "www.not-example.com", exp, "valid")
 
 	// Check that we fail if the CSR signature is invalid
-	ExampleCSR.Signature[0]++
-	certRequest := core.CertificateRequest{
-		CSR: ExampleCSR,
-	}
-
-	_, err := ra.NewCertificate(ctx, certRequest, Registration.ID, 1)
-	ExampleCSR.Signature[0]--
+	ExampleCSR.Raw[len(ExampleCSR.Raw)-1]++
+	fmt.Println(ExampleCSR.Subject.CommonName)
+	fmt.Println(ExampleCSR.DNSNames)
```

Comment:
I think you might have left these in on accident.

Reply:
yup, thanks!